src/syntax-highlighters/monaco/providers/completion.ts

```diff
@@ -0,0 +1,406 @@
+/* eslint-disable max-len */
+
+import type * as MonacoEditor from "monaco-editor";
+import type { editor, languages, Position } from "monaco-editor";
+
+import { SINGLE_INDENTATION } from "../../../formatter/indent-dsl";
+import { Keyword } from "../../../constants/keyword";
+import { assertNever } from "../../../inner-utils/assert-never";
+import { SchemaVersion } from "../../../constants/schema-version";
+
+const provideCompletionItemsOneDotZero =
+  (monaco: typeof MonacoEditor) =>
+  (model: editor.ITextModel, position: Position): languages.CompletionList => {
+    const word = model.getWordUntilPosition(position);
+    const range = {
+      startLineNumber: position.lineNumber,
+      endLineNumber: position.lineNumber,
+      startColumn: word.startColumn,
+      endColumn: word.endColumn,
+    };
+
+    if (position.column === 2) {
```

Comment:
will switch/case be better choice?

Reply:
Switch doesn't work as well, because some of these don't use `===`